go.mod

```diff
@@ -0,0 +1,64 @@
+module github.com/libp2p/go-libp2p-gorpc
+
+require (
+	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412 // indirect
+	github.com/btcsuite/btcd v0.0.0-20190115013929-ed77733ec07d // indirect
+	github.com/coreos/go-semver v0.2.0 // indirect
+	github.com/fd/go-nat v1.0.0 // indirect
+	github.com/google/uuid v1.1.0 // indirect
+	github.com/gorilla/websocket v1.4.0 // indirect
+	github.com/ipfs/go-cid v0.9.0 // indirect
+	github.com/ipfs/go-ipfs-util v1.2.8 // indirect
+	github.com/ipfs/go-log v1.5.7
+	github.com/jbenet/go-temp-err-catcher v0.0.0-20150120210811-aac704a3f4f2 // indirect
+	github.com/jbenet/goprocess v0.0.0-20160826012719-b497e2f366b8 // indirect
+	github.com/libp2p/go-addr-util v2.0.7+incompatible // indirect
+	github.com/libp2p/go-buffer-pool v0.1.3 // indirect
+	github.com/libp2p/go-conn-security v0.1.15 // indirect
+	github.com/libp2p/go-conn-security-multistream v0.1.15 // indirect
+	github.com/libp2p/go-flow-metrics v0.2.0 // indirect
+	github.com/libp2p/go-libp2p v6.0.29+incompatible
+	github.com/libp2p/go-libp2p-autonat v0.0.0-20181221182007-93b1787f76de // indirect
+	github.com/libp2p/go-libp2p-circuit v2.3.2+incompatible // indirect
+	github.com/libp2p/go-libp2p-crypto v2.0.1+incompatible // indirect
+	github.com/libp2p/go-libp2p-discovery v0.0.0-20181218190140-cc4105e21706 // indirect
+	github.com/libp2p/go-libp2p-host v3.0.15+incompatible
+	github.com/libp2p/go-libp2p-interface-connmgr v0.0.21 // indirect
+	github.com/libp2p/go-libp2p-interface-pnet v3.0.0+incompatible // indirect
+	github.com/libp2p/go-libp2p-loggables v1.1.24 // indirect
+	github.com/libp2p/go-libp2p-metrics v2.1.7+incompatible // indirect
+	github.com/libp2p/go-libp2p-nat v0.8.8 // indirect
+	github.com/libp2p/go-libp2p-net v3.0.15+incompatible
+	github.com/libp2p/go-libp2p-peer v2.4.0+incompatible
+	github.com/libp2p/go-libp2p-peerstore v2.0.6+incompatible
+	github.com/libp2p/go-libp2p-protocol v1.0.0
```

Comment:
why is this not indirect. shouldn't we just have a direct import being top-level libp2p?

Reply:
I don't follow. the protocol import?